airflow/utils/log/es_task_handler.py

```diff
@@ -54,7 +55,8 @@ class ElasticsearchTaskHandler(FileTaskHandler, LoggingMixin):
     def __init__(self, base_log_folder, filename_template,
                  log_id_template, end_of_log_mark,
                  write_stdout, json_format, json_fields,
-                 host='localhost:9200'):
+                 host='localhost:9200',
+                 es_kwargs=conf.getsection("elasticsearch_configs") or {}):
```

Comment:
Can we add this to `https://github.com/apache/airflow/blob/master/airflow/config_templates/default_airflow.cfg` as well, please?